packages/block-editor/src/hooks/anchor.js

```diff
@@ -71,8 +71,7 @@ function BlockEditAnchorControlPure( { anchor, setAttributes } ) {
 					<>
 						{ __(
 							'Enter a word or two — without spaces — to make a unique web address just for this block, called an “anchor.” Then, you’ll be able to link directly to this section of your page.'
-						) }
-
+						) }{ ' ' }
```

Comment:
To avoid a space at the end of the text when `isWeb` is `false`, that is, in the mobile app, how about doing the following?

```javascript
help={
	<>
		{ __(
			'Enter a word or two — without spaces — to make a unique web address just for this block, called an “anchor.” Then, you’ll be able to link directly to this section of your page.'
		) }
		{ isWeb && (
			<>
				{ ' ' }
				<ExternalLink
					href={ __(
						'https://wordpress.org/documentation/article/page-jumps/'
					) }
				>
					{ __( 'Learn more about anchors' ) }
				</ExternalLink>
			</>
		) }
	</>
}
```